backend/src/apiserver/server/job_server.go

```diff
@@ -403,33 +399,38 @@ func (s *JobServer) canAccessJob(ctx context.Context, jobID string, resourceAttr
 		// Skip authorization if not multi-user mode.
```

Comment:
Shall we also verify that `namespace` is empty if we are not in multi-user mode?

Reply:
IMO, we should not. I see a standalone deployment of KFP as a mode where a user has access to everything. So specifying a `Namespace` is still a valid request, but we ignore it.